board/safety/safety_hyundai.h

```diff
@@ -35,6 +35,10 @@ AddrCheckStruct hyundai_legacy_rx_checks[] = {
 };
 const int HYUNDAI_LEGACY_RX_CHECK_LEN = sizeof(hyundai_legacy_rx_checks) / sizeof(hyundai_legacy_rx_checks[0]);
 
+const int HYUNDAI_PARAM_EV_GAS = 1;
+const int HYUNDAI_PARAM_HEV_GAS = 2;
```

Comment:
```suggestion
const int HYUNDAI_PARAM_HYBRID_GAS = 2;
```
let's be consistent with this convention otherwise it'll get confusing: hybrid for PHEVs and HEVs and EV for full electric